Animations/Fade/DownBig.js

```diff
@@ -1,30 +1,26 @@
-var Animation = require('../Animations.js');
+var makeAnimation = require('../../src/TuxAnimations');
+var objectAssign = require('object-assign');
```

Comment:
In the rest of our files we set the result of `require('object-assign')` equal to `assign` and use that variable name.  To be consistent wish should do so here as well.